apps/hash-ai-worker-py/README.md

```diff
@@ -0,0 +1,44 @@
+# HASH AI worker (Python)
+
+## Requirements
+
+- Python 3.x > 3.10 (the version used in HASH is 3.11)
```

Comment:
Can you change this, and the other one in `hash-agents` to say 3.11? I think we actually just set that in poetry and it's what we use in CI and such so..